docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+were created by the `network.sh` script:
+```
+docker ps -a
+```
+
+Each node and user that interacts with a Fabric network needs to belong to an
+organization that is a network member. The group of organizations that are
+members of a Fabric network are often referred to as the consortium. The test
+network has two consortium members, `org1.example.com` and `org2.example.com`.
+The network also includes one orderer organization, `example.com`, who maintains
+the ordering service of the network.
+
+Peer are the fundamental components of any Fabric network. Peers store the
+blockchain ledger and validate transactions before they are committed to the
+ledger. Peers run the smart contracts that encode the business logic that is used
+to manage the assets on the blockchain ledger.
+
+Every peer in the network needs to belong to a member of the consortium. In the
+test network, each organization operates one peer each, `peer0.org1.example.com`
+and `peer0.org2.example.com`.
+
+Every Fabric network also includes an ordering service that is the common biding
+of the network. While peers validate transactions and add blocks of transactions
+to the blockchain ledger, they do not decide on the order of transactions or
+include them into new blocks. On a distributed network, peers may be running far
+away from each other and not have a common view of when a transaction was created.
+Coming to consensus on the order of transactions is a costly process that would
+create overhead for the peers.
+
+An ordering service allows peers to focus on validating transactions and
+committing them to the ledger. After they receive transactions from peers,
+ordering nodes come to consensus on the order of transactions and then add them
+to blocks. The blocks are then distributed to peer nodes, which then add the blocks
+the blockchain ledger. Ordering nodes also operate the system channel that
+defines the capabilities of a Fabric network, such as how blocks are made and
+which version of Fabric that nodes can use. The system channel defines which
+organizations are members of the consortium.
+
+The sample network uses a single node Raft ordering service that is operated by
+the ordering organization. You can see the ordering node running on your machine
+as `orderer.example.com`. While the test network only uses a single node ordering
+service, a real network would have multiple ordering nodes, operated by one or
+multiple orderer organizations. The different ordering nodes would use the Raft
+consensus algorithm to come to agreement on the order of transactions across
+the network.
+
+## Creating a channel
+
+Now that we have peer and orderer nodes running on our machine, we can use the
+nodes to create a Fabric channel. Channels are a private layer of communication
+that can be used by only certain network members. Channels can be used only by
+organizations that are invited the channel, and are invisible to other members
```

Comment:
Channels are private but even more importantly they're where transactions happen (application channels, specifically). A beginner might think that creating nodes and having endpoints is enough. The channel is the context in which any transaction will occur.

Reply:
I am going to use this instead "Now that we have peer and orderer nodes running on our machine, we can use the script to create a Fabric channel for transactions between Org1 and Org2. Channels are a private layer of communication.."